Purchases/SwiftSources/Purchasing/Transaction.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  Transaction.swift
+//  Purchases
+//
+//  Created by RevenueCat.
+//  Copyright © 2020 Purchases. All rights reserved.
+//
+
+import Foundation
+
+@objc(RCTransaction) public class Transaction: NSObject {
+
+    let revenuecatId: String
+    let productId: String
+    let purchaseDate: Date
+
+    internal init(transactionId: String, productId: String, purchaseDate: Date) {
+        self.revenuecatId = transactionId
+        self.productId = productId
+        self.purchaseDate = purchaseDate
+        super.init()
+    }
+
+    internal init(with data: Dictionary<String, Any>, productId: String, dateFormatter: DateFormatter) {
```

Comment:
`Dictionary<String, Any>` -> `[String: Any]`

Reply:
could we rename `data` to something more descriptive of what it should contain? 
developers could also expect `data` to be of type `Data` and not a dict

Reply:
what do you think about `serverResponse`?

Reply:
sounds great